test/mri/excludes/TestNum2int.rb

```diff
@@ -7,3 +7,5 @@
 exclude :test_num2uint, "needs investigation"
 exclude :test_num2ulong, "needs investigation"
 exclude :test_num2ushort, "needs investigation"
+exclude :test_num2ll, "needs investigation"
+exclude :test_num2ull, "needs investigation"
```

Comment:
Could you split these changes in another commit, or maybe just reuse this one but rename and remove the tool?

Reply:
Done. Tool removed and message updated.